services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
```

Comment:
The `create_movie_session` function uses `Movie.objects.get(id=movie_id)`, which will raise a `DoesNotExist` exception if the movie ID is not found. Consider handling this exception or using `filter().first()` to avoid runtime errors.